runhouse/resources/secrets/provider_secrets/ssh_secret.py

```diff
@@ -75,7 +75,7 @@ def _write_to_file(
         priv_key_path = Path(os.path.expanduser(priv_key_path))
         pub_key_path = Path(f"{os.path.expanduser(priv_key_path)}.pub")
 
-        if priv_key_path.exists() and pub_key_path.exists():
+        if priv_key_path.exists() or pub_key_path.exists():
```

Comment:
don't we still need both tho for launching?

Reply:
yup, I just wanted to avoid overwriting in the off chance that one is saved and one isn't but actually it makes more sense to leave this as `and` and address them individually after